pkg/kudoctl/packages/install/package.go

```diff
@@ -139,3 +144,22 @@ func validateParameters(instance v1beta1.Instance, parameters []v1beta1.Paramete
 
 	return nil
 }
+
+func updateOperatorTasks(pkgs []Dependency, operatorVersion *v1beta1.OperatorVersion) {
+	tasks := operatorVersion.Spec.Tasks
+
+	for i := range tasks {
```

Comment:
This is just for top-level tasks, right? But what if we have `./top-level-operator` referencing `./child-level-operator`?